src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/RasterLayerGeoPackage/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "Raster layer (GeoPackage)",
+  "SampleName": "RasterLayerGeoPackage",
+  "Description": "This sample demonstrates how to open a GeoPackage and show a GeoPackage raster in a raster layer.",
+  "Instructions": "The GeoPackage will be downloaded from an ArcGIS Online portal automatically.",
+  "Type": 0,
+  "RequiresOnlineConnection": true,
+  "RequiresOfflineData": true,
+  "DataItemIds": [ "68ec42517cdd439e81b036210483e8e7" ],
```

Comment:
The following line of XML text needs to be added after the "DataItemsIds" line:
```
"DataFileNames": [ "AuroraCO.gpkg" ],
```
This was something new that Nathan added as part of the DataManager for the LibRefAPI documentation builds.